setup.py

```diff
@@ -32,6 +32,12 @@
                     if not line.startswith('#')]
 
 
+extras_require = {
+    'test fixtures': ['caproto', 'curio', 'pytest >=3.9', 'trio'
+                      'git+git://github.com/NSLS-II/event-model@master']
```

Comment:
For whatever reason, `setup.py` can't handle git URLs the way that `pip` + `requirements-dev.txt` can. (That's actually the reason that I made a release -- so we could specify it in `setup.py`/`requirements.txt`.)

But since this is always a hard dependency (in `requirements.txt` and thus in `install_requires`) we don't have to include it here.

Reply:
also fixed